source/includes/_dataset_resources.md

```diff
@@ -0,0 +1,276 @@
+# Dataset Resources
+
+## The resource object
+
+The resource object describe a resource on the OpenDataSoft platform. It is composed of a URL, a title, a type, a parameter object and a credentials object. The URL is where data will be pulled to populate the dataset. Resources urls can (and often do!) point to [files](#files) uploaded to the platform using the `odsfile://` url scheme.
+
+### Attributes
+
+> Example object
+
+```json
+{
+    "resource_uid": "re_abcdef",
+    "url": "odsfile://resource.csv",
+    "title": "My Awesome Data File",
+    "type": "csvfile",
+    "params": {
+        "headers_first_row": false,
+        "separator": ";"
+    },
+    "credentials": {},
+}
+```
+
+Attribute | Description
+--------- | -----------
+`resource_uid` <br> *string* | Unique identifier for the resource
+`url` <br> *string* | URL of the resource
+`title` <br> *string* | friendly title
+`type` <br> *string* | extractor type that should handle this resource
+`params` <br> *object* | parameters passed to the extractor
+`credentials` <br> *object* | resource credentials
+
+## List dataset resources
+
+This endpoint is meant to list all resources that are linked to a dataset.
+
+> Definition
+
+```HTTP
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/resources/
```

Comment:
Isn't that supposed to be `/api/management/v2/{dataset_uid}/resources/`? (same for all examples below)

Reply:
:+1: